packages/rolldown/src/utils/bindingify-input-options.ts

```diff
@@ -279,16 +279,19 @@ function bindingifyTreeshakeOptions(
   if (config === false) {
     return undefined
   }
+
   if (config === true || config === undefined) {
     return {
       moduleSideEffects: true,
       annotations: true,
     }
   }
+
   let normalizedConfig: BindingInputOptions['treeshake'] = {
     moduleSideEffects: true,
     annotations: config.annotations ?? true,
     manualPureFunctions: config.manualPureFunctions,
+    unknownGlobalSideEffects: config.unknownGlobalSideEffects ?? true,
```

Comment:
The [rust side](https://github.com/rolldown/rolldown/commit/c764388bfb8751cf457dacf230d05ef87f4fd67a#diff-fef0515e8da9bf5d47508ef403741c46e31b0e52647ad29efd4c037df8e42ecbR301) already set default value, here `true` could be removed.

Reply:
nice catch!